mimic/model/ironic_objects.py

```diff
@@ -170,6 +175,8 @@ def detail_json(self):
         if self.instance_uuid:
             template["instance_info"] = self.static_instance_info
             template["provision_state"] = "active"
+        if not self.driver:
+            template["driver"] = "agent_ipmitool"
```

Comment:
Not 100% sure the semantics here, but might could be moved up (see comment above)